src/renderer/containers/widgets/createUsernameModal/CreateUsername.tsx

```diff
@@ -18,27 +19,33 @@ const CreateUsernameModal = () => {
 
   const id = useSelector(identity.selectors.currentIdentity)
   const certificate = useSelector(identity.selectors.currentIdentity)?.userCertificate
-
   const communityErrors = useSelector(errors.selectors.currentCommunityErrorsByType)
   const error = communityErrors?.[socketActionTypes.REGISTRAR]
 
+  const invitationUrl = useSelector(communities.selectors.registrarUrl)
+  const channels = useSelector(publicChannels.selectors.publicChannels)
+  const numberOfChannels = channels.length
```

Comment:
It's a little bit weird to make another const out of selector value. I think using channels.length would be enough